packages/bella/src/back.js

```diff
@@ -21,8 +21,8 @@ export default function (part) {
   points.cbNeckCp1 = new Point(points.hps.x * 0.8, points.cbNeck.y)
   let slope = measurements.shoulderSlope * options.shoulderSlopeBack * -1
   points.shoulder = utils.beamsIntersect(
-    new Point((measurements.shoulderToShoulder * options.shoulderToShoulderCorrection) / 2, 0),
-    new Point((measurements.shoulderToShoulder * options.shoulderToShoulderCorrection) / 2, 100),
+    new Point((measurements.shoulderToShoulder * (1+options.shoulderToShoulderEase)) / 2, 0),
+    new Point((measurements.shoulderToShoulder * (1+options.shoulderToShoulderEase)) / 2, 100),
```

Comment:
```suggestion
    new Point((measurements.shoulderToShoulder * (1 + options.shoulderToShoulderEase)) / 2, 0),
    new Point((measurements.shoulderToShoulder * (1 + options.shoulderToShoulderEase)) / 2, 100),
```

Reply:
Isn't there some formatting process that makes everything uniform? I bothered doing so quite a while ago (since it doesn't like my code formatting preferences ;-)

Reply:
> Isn't there some formatting process that makes everything uniform? I bothered doing so quite a while ago (since it doesn't like my code formatting preferences ;-)

In Visual Studio Code you can run "Format Document". In this file it will make the suggested changes.

On Mac you can accomplish this with any of the following:

- To format an entire file, while in the file type ⇧ ⌥ F
- Alternatively, type ⇧ ⌘ P then start typing "format document" and hit enter once it autocompletes.
- To format a highlighted selection, highlight some code and then type: ⌘ K, then ⌘ F
- Alternatively, type ⇧ ⌘ P then start typing "format selection" and hit enter Enter once it autocompletes.

Reply:
Or run `yarn prettier` in the monorepo root :)

Reply:
I grepped for "format" before posting this. Didn't know about "prettier"! Thanks!

Reply:
That's why I stopped bothering with formatting. I have prettier integrated with VS Code (my preferred editor), but my settings are different ;-)